src/hockeypuck/server/settings.go

```diff
@@ -244,36 +252,51 @@ func DefaultSettings() Settings {
 }
 
 func ParseSettings(data string) (*Settings, error) {
-	// Parse the configuration file as a template first
-	tmpl, err := template.New("config").Funcs(sprig.TxtFuncMap()).Funcs(envFuncMap()).Parse(data)
-	if err != nil {
-		return nil, errors.WithStack(err)
+	// Check if data contains template syntax - if so, process as template first
```

Comment:
Is there any harm just processing as a template anyway? Tiny bit of latency on startup, but surely only microseconds...?